src/components/panes/TargetPane.scss

```diff
@@ -63,6 +77,28 @@
         }
     }
 
+    .TargetPane-personField {
+        &:before {
+            @include icon($fa-var-list);
```

Comment:
I'm not a big fan of this icon repeated over and over again. Maybe something like `$fa-var-info-circle` instead?